lib/coarse_op.cu

```diff
@@ -2,14 +2,16 @@
 #include <color_spinor_field.h>
 #include <gauge_field.h>
 #include <clover_field.h>
+
+#define WILSONCOARSE
```

Comment:
add a comment here perhaps to explain what this does

Reply:
Done here, as well as in `lib/coarsecoarse_op.cu` and `lib/staggered_coarse_op.cu`.